back-end/test/getShow.js

```diff
@@ -0,0 +1,23 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const axios = require('axios')
+const sinon = require('sinon')
+const expect = chai.expect
+const server = require('../app.js')
+chai.use(chaiHttp)
+
+describe('GET /show', () => {
+  it('should return 200 OK', async () => {
+    const stub = sinon.stub(axios, 'get').resolves({data: {id: 1, name: 'sample show'}})
+    const res = await chai.request(server).get('/show')
```

Comment:
Looks good Mason! The only thing that needs to be fixed is everywhere there's `/show`, it should be `/shows/1`